### DISCUSSION TITLE
HEIMAN smoke sensor HS1SA-E

### DISCUSSION THREAD

**Discussion by RomanLipa** [Koenkk/zigbee2mqtt#6022] [Devices]
![Zigbee2Mqtt](https://user-images.githubusercontent.com/58612386/106454153-9e5b9a80-648a-11eb-8d4e-90a7039e964e.jpg)
![log](https://user-images.githubusercontent.com/58612386/106454158-9ef43100-648a-11eb-818a-18277b8a614e.jpg)

Can anyone help me in adding a HEIMAN HS1SA-E smoke sensor to Domoticz. I am using a Zigbee CC2531 USB Dongle with the Zigbee2Mqtt plugin.
In the Custom menu Zigbee2mqtt is seen as 0x000d6f00160951c2, Model SMOK_YDLV10N,
but in the log it is written: "2021-01-28 19: 19: 33.634 (Zigbee2MQTT) 0x000d6f00160951c2: device definiton not found, skipping ..."
What could be wrong.
Greetings,
Roman

**Comment by danielschramm** [comment]
Hello,
I have 2 HS1SA:
Zigbee model: SmokeSensor-EM
IEEE address: 0x000d6f00149573de
Firmware build date: 2018.4.20
Firmware version: 1.1.1
Vendor: HEIMAN
Model: HS1SA

and
Zigbee model: SmokeSensor-N-3.0
IEEE address: 0x00158d000275473e
Firmware build date: 2018.9.27
Firmware version: 2000-0001
Vendor: HEIMAN
Model: HS1SA

The first one is working perfectly. 
The second one has some issues completing the interview but is recognizes and sends all values.

Daniel

**Comment by Kat-CeDe** [comment]
Hi,
I also have 3 of them. 2 worked on 1st try and the 3rd must be paired several times but now it looks like he also working normal. Did you check data base/log if interview is completed and configure 1?

Ralf

**Reply by danielschramm** [reply]
Hello Ralf,
I currently did not invest a lot of time into this single device since my more important question is still not solved.
How do I use the smoke event?

https://github.com/Koenkk/zigbee2mqtt/discussions/6001

Do you have any ideas?

Thanks,

Daniel

**Comment by Kat-CeDe** [comment]
Hi,
sorry but there I cannot help. I use IP Symcon and get a boolean for smoke on which I can trigger. I use 1.17.1 but it also worked with 1.16 of Zigbee2MQTT.  Did you look at the Z2M log file and data base if interview was completed and configure:1?

When I had trouble with this 1 device I stopped Z2M, removed the entry from the data base, started Z2M again and paired it again.

As you use the CC2531 how many devices do you have? The CC2531 has quite limited performance and with 20+ devices you might get problems.

Ralf

**Reply by danielschramm** [reply]
Of course I get the Smoke Signal in Z2M, but an alarm system should still forward events from a sensor to siren without Z2M beeing active.
Currently I'm testing with <10 Devices If everything works I might invest in a better interface.

**Comment by Kat-CeDe** [comment]
Hi,
I do not think that a sensor (smoke) can send messages/commands directly to an actor (siren) like Z-Wave can. Afaik some kind of bridge like Z2M and software is always needed.

Ralf
